lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -72,7 +72,8 @@ class OffscreenImages extends Audit {
     const url = URL.getDisplayName(image.src, {preserveQuery: true});
     const totalPixels = image.clientWidth * image.clientHeight;
     const visiblePixels = this.computeVisiblePixels(image.clientRect, viewportDimensions);
-    const wastedRatio = 1 - visiblePixels / totalPixels;
+    // Treat images with 0 area as if they're offscreen
```

Comment:
nit: link to https://github.com/GoogleChrome/lighthouse/issues/1914 if you want and mention that this can happen if the container is `display: none`.